arangod/VocBase/Methods/Collections.h

```diff
@@ -73,7 +73,7 @@ struct Collections {
   static arangodb::Result lookup( // find collection
     TRI_vocbase_t const& vocbase, // vocbase to search
     std::string const& name, // collection name
-    FuncCallback const& callback // invoke on found collection
+    FuncCallback callback // invoke on found collection
```

Comment:
it's usually good practice to use 'const&' with lambdas to avoid making a copy of a lambda